### ISSUE TITLE
Switch 'master' to 'main'

### ISSUE THREAD

**Issue by mattecapu** [zulip/zulip-archive#46]
Since Git(Hub) changed the default branch name to `main` zulip-archive should use that when commiting new changes, I believe [here](https://github.com/zulip/zulip-archive/blob/eb1605d1557c6650cce7f2dc0dad95aca257658a/entrypoint.sh#L78) and in subsequent lines.

Even better would be to allow the user to specify a preferred branch name.

**Comment by rht** [comment]
Right. Here is the thread of current discussion: https://chat.zulip.org/#narrow/stream/2-general/topic/master-slave.20term.
